### PR TITLE
Update Documentation for `\includescore`.

### PR DESCRIPTION
**Pull request by eschwab**
Addresses part of #89

### CODE REVIEW

**Comment by rpspringuel** [comment]
I think it might be better to have a separate section for package options, rather than including the in the User commands section.  I realize that we don't have many options right now, but I could foresee turning all the flag controlled behaviors into options long term, in which case having the separate section would help make the document more clearly organized.

**Comment by eschwab** [comment]
Good idea. Done.

**Comment by rpspringuel** [comment]
Obviously you haven't done (or at least not pushed) the changes related to my most recent modifications, but it might be nice when you do so to include a warning that when using the never compile option (in any scope) the filename given has to be for an existing file and the only extension that can be omitted is `.tex` (since this filename gets passed straight to `\input` which is capable of adding that extension if needed).  This is slightly different than the other two options, where the algorithms can be given any of four different extensions (omitted, `.gtex`, `.gabc`, and `.tex`) and still find the right file.

Or maybe we should modify the code so that the never compile option can handle the same set of extensions?

**Comment by eschwab** [comment]
I don't think that would be necessary. As it stands now the default is old gregoriotex behavior and should not suprise users. If someone uses the `nevercompile` feature it is their responsibility to provide a current `gtex/tex` with correct `gregorio_version` and they should be able to provide a good `\includescore` argument.

Definitly I can make a note as you suggest and also encourage omitting the file extension all together as the best practice.

**Comment by eschwab** [comment]
This should be done now. Br. Samuel, could you proofread and make sure that I got all the arguments and their respective descriptions correct?

File: doc/Command_Index_User.tex [hunk 1]
```diff
@@ -6,37 +6,147 @@ \section{User Controls}
...
+behavior of gregoriotex. It is the package default.
+
+\medskip The three previous macros can be combined in the same document to
+switch between different includescore behaviors: \par\medskip
+\begin{latexcode}
+  \usepackage{gregoriotex} % [nevercompile] is the default.
+  ----
+  \includescore{TecumPrincipium} % gabc never compiled.
+  \includescore[f]{TecumPrincipium} % gabc always compiled.
+  \includescore[a]{TecumPrincipium} % dabc auto compiled.
```
Comment (rpspringuel) [inline doc/Command_Index_User.tex]: typo: `dabc` should be `gabc` in comment

File: doc/Command_Index_User.tex [hunk 2]
```diff
@@ -6,37 +6,147 @@ \section{User Controls}
...
+
+  \autocompilegabc
+  \includescore{TecumPrincipium} % gabc auto compiled.
+  \includescore[n]{TecumPrincipium} % gabc never compiled.
+  \includescore[f]{TecumPrincipium} % gabc always compiled.
+
+  \forcecompilegabc
+  \includescore{TecumPrincipium} % gabc auways compiled.
```
Comment (rpspringuel) [inline doc/Command_Index_User.tex]: typo: `auways` should be `always` in comment

**Comment by rpspringuel** [comment]
Other than those two typos it looks good.  I'll merge once you fix them.

**Comment by eschwab** [comment]
Thank you, and fixed.

**Pull request merged**